test/Interceptor/HstsTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Amp\Http\Client\Interceptor;
+
+use Amp\Http\Client\Interceptor\Hsts\GooglePreloadListJar;
+use Amp\Http\Client\Interceptor\Hsts\HstsInterceptor;
+use Amp\Http\Client\Interceptor\Hsts\InMemoryHstsJar;
+
+class HstsTest extends InterceptorTest
+{
+    public function testHstsHost(): void
+    {
+        $hstsJar = new InMemoryHstsJar();
+        $hstsJar->register("example.org");
+        $this->assertTrue($hstsJar->test("example.org"));
+//        $this->givenApplicationInterceptor(new HstsInterceptor($hstsJar));
```

Comment:
Uncommenting this in makes the test never finish - because the HTTP server for InterceptorTest will not respond to an HTTPS request